app/views/categories/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="items-container rounded-md bg-gray-100  p-3  shadow-sm ">
+<%= form_for @category do |f| %>
+<%= f.text_field :name, placeholder: "Enter Category's name", class:"block all-in-one-label   round-md bg-transparent py-1.5 pl-1  sm:text-sm sm:leading-6 " %>
```

Comment:
- [ ] Dear @fuadmoin, while creating a category, Please make sure the app can stand to end user error, the user might submit the form without filling in the required/mandatory fields, and this is causing an issue in your case. Your app crashes in such a case. Please apply HTML5 form validation to make these fields required and overcome the issues. 

### Submitting form without filling it 
![image](https://github.com/fuadmoin/all-in-one/assets/69077061/e5c40b1a-af3e-46f1-a0e5-c41ced71122e)

- [ ] These same issues happen on the `Add a new transaction/item`, Please take a look at this too.  And here, its mandatory to check at least one `category`, beside the others.
> It's recommended we always validate all user input to make sure that anyone with bad intentions cannot compromise our app, both in front and backend.